packages/ui-react/src/components/VideoMetaData/VideoMetaData.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import styles from './VideoMetaData.module.scss';
+
+type Props = {
+  attributes: unknown[];
+  separator?: string;
+};
+
+const VideoMetaData: React.FC<Props> = ({ attributes, separator = '•' }: Props) => {
+  return (
+    <>
+      {attributes.map((value, index) => (
+        <>
+          <span key={index}>{value as string}</span>
```

Comment:
```suggestion
          <span key={index}>{value}</span>
```

Reply:
Fixed. See comment above. I will resolve this one